index.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jery Mey Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="body-white">
+        <nav class="navbar">
+            <div class="logo">
+              <a href="#">My Logo</a>
+            </div>
+            <div class="menu-icon">
+                <img src="images/appbar.png" alt="menu button" />
+            </div>
+            <ul class="nav-links">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+       <section class="header">
+        <h1 class="title">I’m Jery Mey. <br>
+            Glad to see <br> you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <a class="link" href="#">LET'S CONNECT</a>
+        <ul class="social-icons">
+           <li><a href=""><img src="images/social-media-icon/tw.png" alt="twitter"></a></li>
+           <li><a href=""><img src="images/social-media-icon/linkedin.png" alt="linkedin"></a></li>
+           <li><a href=""><img src="images/social-media-icon/medium.png" alt="medium"></a></li>
+           <li><a href=""><img src="images/social-media-icon/github.png" alt="github"></a></li>
+           <li><a href=""><img src="images/social-media-icon/v.png" alt="v"></a></li>
+        </ul>
+
+      </section>
+      <!-- ---------- cards section ----------->
+
+      <section class="skills">
+        <article class="cards">
+            <figure>
+                <img src="images/tonic.png" alt="">
+            </figure>
+            <h2 class="title-card">Tonic</h2>    
+        <div class="header-details">
+            <h4>CANOPY</h4>           
+            <ul class="dot">              
+                <li>Back End Dev</li>               
+                <li>2015</li>    
+            </ul>
+        </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="skills-card">
+                <li class="skill-1">html</li>
+                <li class="skill-1">css</li>
+                <li class="skill-1">Javascript</li>
+            </ul>
+            <button class="btn"><a href="">See Project</a></button>
+        </article>
+
```

Comment:
Good job creating the card to show projects in your portfolio, however for semantic reasons it could be betetr to have all the cards in a `section` container 👍 , as it will aslo help you to meet the requirement that says that the main container of the projects should be a display:grid 😄  Kindly wrap your cards in a container and make sure you use dispay:grid for the container 👍